### ISSUE TITLE
Disk_info not working correctly on Snow Leopard

### ISSUE THREAD

**Issue by Steffan-Ravn** [munkireport/munkireport-php#481]
The script disk_info in the disk_report module does not correctly gather disk data on Mac OS X 10.6 Snow Leopard, as the output of 
`diskutil list -plist`
differs from later versions of diskutil on newer OS X versions.

I have made some changes in my disk_info script locally to get correct information, but as I am no Python-expert, I imagine it can be handled more elegantly. Basically I have created an extra function (or definition) to handle diskutil on Snow Leopard, so the output mimics the one created on Lion or later versions.

```
def _DictFromDiskutilList_SnowLeopard():
    """as Snow Leopard delivers a different output from diskutil 
    list we will mimic the output from newer versions of OS X"""
    Partitions = _DictFromDiskutilList()["AllDisks"]
    DiskArray = []

    for Partition in Partitions:
        DiskArray.append(_DictFromDiskutilInfo(Partition))

    plist = [{'Content':'Guid', 'Partitions':DiskArray,}]
    plistfile = '%s/cache/SL_tmpdisk.plist' % os.path.dirname(os.path.realpath(__file__))
    plistlib.writePlist(plist, plistfile)
    return plistlib.readPlist(plistfile)
```

I have then altered the AllDisksAndPartitions definition to check for os-version and then run either the Snow Leopard version of the function or the standard function. 

I've attached my full disk_info script for comparison.
[disk_info.zip](https://github.com/munkireport/munkireport-php/files/256927/disk_info.zip)

**Comment by bochoven** [comment]
Ok, nice solution. Do you want to file a Pull Request for this?

**Comment by Steffan-Ravn** [comment]
Thanks - I've created a Pull Request. :)

**Comment by bochoven** [comment]
Fixed with 733e224d0d3110b704d736c89dd3c4490b5336c9
